index.html

```diff
@@ -21,13 +21,25 @@
                 <li><a href="#">Contact</a></li>
                 <i class="fa-solid fa-envelope"></i>
             </ul>
+            <div class="dropdown-menu hide" id="myDropdown">
+                <span class="xmark"><i class="fa-solid fa-xmark"></i></span>
```

Comment:
- [ ] As requested by the previous reviewer, Please reduce the height(visibility) of the bottom border for mobile menu options.

### Your design

![image](https://user-images.githubusercontent.com/83803414/208877264-4e28852c-b029-438b-b0ea-10b8665275a0.png)

### Figma design

![image](https://user-images.githubusercontent.com/83803414/208877420-340d11d9-2301-423e-ba67-1bbb414e7acd.png)